### ISSUE TITLE
Make hover over of files display equinor.com instead of sanity.io

### ISSUE THREAD

**Issue by fernandolucchesi** [equinor/energyvision#1244]

**Comment by SvSven** [comment]
Need to contact Sanity to set up alias for CDN links.
Not sure how involved the process is, but it is something they support.

**Comment by fernandolucchesi** [comment]
Follow: https://equinor.slack.com/archives/C01A4BGNRMJ/p1661942713783609

**Comment by fernandolucchesi** [comment]
We need to add these DNS records:

```
cdn.equinor.com                  IN CNAME    cdn.sanity-host.com
h61q9gi9.api.equinor.com         IN CNAME    api.sanity-host.com
h61q9gi9.apicdn.equinor.com      IN CNAME apicdn.sanity-host.com
```

@padms, can you please handle the RITM ticket?

**Comment by padms** [comment]
RITM2583605, RITM2583601, RITM2583593

**Comment by padms** [comment]
CNAME entry added. Not sure if we should request for SSL certificate, as the server is hosted by sanity.

**Comment by SvSven** [comment]
We need to remember to add cdn.equinor.com to the `next.config` and security headers like we did for cdn.sanity.io

**Comment by SvSven** [comment]
Looks good to me, is working for me locally as well.

Will need to update here as well for release: https://github.com/equinor/energyvision/blob/main/web/pageComponents/shared/Hero/Hero5050.tsx#L70

We should probably also have this documented somewhere, so we know how/what/why and stuff.

**Comment by fernandolucchesi** [comment]
@meols can you please review this? https://web-global-test-equinor-web-sites-test.c2.radix.equinor.com/

For other domains .br .de .jp etcs it will also show equinor.com

**Comment by meols** [comment]
@fernandolucchesi - the hover on files looks great - but is this also affecting images in some way? I don't see any images on the site: ![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/89cc480e-38bf-4bda-ae24-6af2ac2d769b)

**Comment by meols** [comment]
I tried to open an image in a new tab and got this: ![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/858a807e-ea04-4075-a014-9d5044786a0b)

**Comment by fernandolucchesi** [comment]
> @fernandolucchesi - the hover on files looks great - but is this also affecting images in some way? I don't see any images on the site: ![image.png](https://camo.githubusercontent.com/4c0f6d6326af915ce3e42d90783f58a4c14a8f1ea1a9787b424fe708db4c0536/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3566626266346632626162386165373434636236353630662f38396363343830652d333862662d346264612d616532342d366166326163326437363962)

are you using equinor's ethernet / wifi? We might need them to unblock the new url "cdn.equinor.com". @padms do you remember where we opened the request last time?

**Comment by fernandolucchesi** [comment]
> @fernandolucchesi - the hover on files looks great - but is this also affecting images in some way? I don't see any images on the site: ![image.png](https://camo.githubusercontent.com/4c0f6d6326af915ce3e42d90783f58a4c14a8f1ea1a9787b424fe708db4c0536/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3566626266346632626162386165373434636236353630662f38396363343830652d333862662d346264612d616532342d366166326163326437363962)

On my end:

<img width="779" alt="image" src="https://user-images.githubusercontent.com/23337266/193060145-1af53949-99fd-435b-970a-57a5a2bced0a.png">

**Comment by meols** [comment]
Yes - I am at the Fornebu office today @fernandolucchesi

**Comment by meols** [comment]
![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/5e4a028b-ea66-412c-9790-303c0d8848b8)

**Comment by meols** [comment]
Great! :thumbsup:

**Comment by fernandolucchesi** [comment]
https://equinor.service-now.com/selfservice?id=ticket&table=sc_req_item&sys_id=0c322bf087521590e70032e83cbb35cb

**Comment by fernandolucchesi** [comment]
@meols, can you please test this one again?

**Comment by meols** [comment]
@fernandolucchesi - working at the Porsgrunn office today - images still aren't visible. 
![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/936ef460-97bb-444c-878c-2e2be0da8a89)

**Comment by fernandolucchesi** [comment]
> https://web-global-test-equinor-web-sites-test.c2.radix.equinor.com/

Reopened a new ticket...
https://equinor.service-now.com/selfservice?id=ticket&table=sc_req_item&sys_id=9c971e0087e619d01b6564670cbb35e2

**Comment by meols** [comment]
@padms - are there any updates to these requests? If not, can you please ask for an update and an estimate for when they will be solved?

**Comment by SvSven** [comment]
@fernandolucchesi @padms Are there any updates here? It looks to be working without errors now when I'm at the office

**Comment by padms** [comment]
Yes the ticket is closed in ServiceNow

**Comment by meols** [comment]
@fernandolucchesi @padms - images are showing on Equinor network (tested from home on VPN) and hover over is showing equinor.com instead of sanity - **except** within a table ( see [crude oil assay](https://web-global-test-equinor-web-sites-test.c2.radix.equinor.com/energy/crude-oil-assays) table). Is that expected? ![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/52acdf0d-775c-4840-af5f-1d4355f801f0)

**Comment by fernandolucchesi** [comment]
> @fernandolucchesi @padms - images are showing on Equinor network (tested from home on VPN) and hover over is showing equinor.com instead of sanity - **except** within a table ( see [crude oil assay](https://web-global-test-equinor-web-sites-test.c2.radix.equinor.com/energy/crude-oil-assays) table). Is that expected? ![image.png](https://camo.githubusercontent.com/6c9145f3ca0aa6eeefc40e266e68d6daf35fd349573285331221facce72bd49d/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3566626266346632626162386165373434636236353630662f35326163646630642d373735632d343834302d616635662d316434333535663830316630)

should be fixed now :)

**Comment by meols** [comment]
@fernandolucchesi - thanks! Then we're good :smile:
